qa/tasks/cephfs/filesystem.py

```diff
@@ -617,7 +617,7 @@ def required_client_features(self, *args, **kwargs):
 
     # In Octopus+, the PG count can be omitted to use the default. We keep the
     # hard-coded value for deployments of Mimic/Nautilus.
-    pgs_per_fs_pool = 8
+    pgs_per_fs_pool = 64
```

Comment:
8 pgs is indeed to few. The poor distribution is probably affecting other tests even. However, as the comment indicates, there's no need to hardcode these values anymore. We don't use this version of the qa/ suite with Nautilus or older. So we can even remove these hard coded values. Instead, we should let Ceph pick a default and then adjust it using new-style pg settings. Namely: `ceph osd pool set $DATA_POOL_NAME target_size_ratio 0.9`. Then the autobalancer will adjust it promptly. You can also do: `ceph osd pool set $DATA_POOL_NAME pg_num_min 64` to ensure that the autobalancer does not later reduce it below that number. You should do the same (pg_num_min, but not target_size_ratio) for the metadata pool.

Reply:
One last thing: if the test is using EC pools then we'll want to use a lower `target_size_ratio` for the default replicated data pool (maybe `.05`) and a higher value for the EC pool.

Reply:
See also:

https://ceph.io/rados/new-in-nautilus-pg-merging-and-autotuning/

and

https://docs.ceph.com/en/latest/rados/operations/placement-groups/#specifying-expected-pool-size

Reply:
Yeah, this sounds nice, I will do it.